packages/cli/cloud/src/environment/command.ts

```diff
@@ -1,7 +1,12 @@
 import { Command } from 'commander';
 import { defineCloudNamespace } from '../cloud/command';
 
-export function createEnvironmentCommand(command: Command): Command {
-  const cloud = defineCloudNamespace(command);
-  return cloud.command('environment').description('Manage environments for a Strapi Cloud project');
-}
+let environmentCmd: Command | null = null;
+
+export const getEnvironmentCommand = (command: Command): Command => {
```

Comment:
"get" or "create" are okay, if you're looking for other suggestions: `setup` or `initialize` could be more explicit (at least to me)

Reply:
The function will be used in other environment subcommands. Using `create` suggests that we are creating a new command each time it's used, which may be misleading. I believe `get` better conveys the idea of retrieving an existing instance, but I’m leaning now toward `initialize`.